netconify.txt

```diff
@@ -1 +0,0 @@
-importlib>=1.0.3; python_version == '2.6'
```

Comment:
Maybe I'm mis-interpreting the diff output, but it looks like you've removed the only line in the file, but haven't removed the file itself. It doesn't look like the file is needed anymore, so can we remove the file itself?